docs/topics/Permissions.md

```diff
@@ -59,7 +59,7 @@ Below is a table of all current permissions, their integer values in hexadecimal
 | MANAGE_ROLES \*               | `0x0010000000` `(1 << 28)` | Allows management and editing of roles                                                                                             | T, V, S      |
 | MANAGE_WEBHOOKS \*            | `0x0020000000` `(1 << 29)` | Allows management and editing of webhooks                                                                                          | T            |
 | MANAGE_EMOJIS_AND_STICKERS \* | `0x0040000000` `(1 << 30)` | Allows management and editing of emojis and stickers                                                                               |              |
-| USE_SLASH_COMMANDS            | `0x0080000000` `(1 << 31)` | Allows members to use slash commands in text channels                                                                              | T            |
+| USE_SLASH_COMMANDS            | `0x0080000000` `(1 << 31)` | Allows members to use application commands, including slash commands and context menu commands.                                    | T            |
```

Comment:
@msciotti Should this be changed to `USE_APPLICATION_COMMANDS` since the name doesn't fit anymore?

Reply:
Yes. The permission in our backend was actually always called `USE_APPLICATION_COMMANDS`. Not sure why it doesn't say that here.